tests/layer_tests/common/mo_convert_test_class.py

```diff
@@ -33,19 +38,21 @@ def _test(self, temp_dir, test_params, ref_params):
         assert not exit_code, (
             "Reference IR generation failed with {} exit code: {}".format(exit_code, stderr))
 
-        ir_test = IREngine(Path(temp_dir, 'model_test.xml'), Path(temp_dir, 'model_test.bin'))
-        ir_ref = IREngine(Path(temp_dir, 'model_ref.xml'), Path(temp_dir, 'model_ref.bin'))
-        flag, resp = ir_test.compare(ir_ref)
-        assert flag, '\n'.join(resp)
+        ir_test = core.read_model(Path(temp_dir, 'model_test.xml'))
+        ir_ref = core.read_model(Path(temp_dir, 'model_ref.xml'))
+
+        flag, msg = compare_functions(ir_test, ir_ref)
+        assert flag, '\n'.join(msg)
 
     def _test_by_ref_graph(self, temp_dir, test_params, ref_graph):
         """
         Generates IR using MO Python API, reads it and compares with reference graph.
         """
-        test_params.update({"model_name": 'model_test', "output_dir": temp_dir})
+        from openvino.runtime import Core
+        core = Core()
 
+        test_params.update({"model_name": 'model_test', "output_dir": temp_dir})
         self.generate_ir_python_api(**test_params)
-
-        ir_test = IREngine(Path(temp_dir, 'model_test.xml'), Path(temp_dir, 'model_test.bin'))
-        flag, resp = ir_test.compare(ref_graph)
-        assert flag, '\n'.join(resp)
+        ir_test = core.read_model(Path(temp_dir, 'model_test.xml'))
```

Comment:
Do you clean-up all generated-by-test files like `model_test.xml` and so? I have not found it. Please check.

Reply:
Removing of temp_dir is handled by tempfile lib. It is initialized here https://github.com/openvinotoolkit/openvino/blob/f893a58816c8c84e91541f5ba9f249154d59dfa2/tests/layer_tests/conftest.py#L105
tempfile lib removes the temporary dir.